opencti-platform/opencti-front/src/private/components/cases/case_incidents/IncidentKnowledgeCorrelation.jsx

```diff
@@ -389,6 +390,29 @@ class IncidentKnowledgeCorrelationComponent extends Component {
     });
   }
 
+  handleToggleQueryMode() {
```

Comment:
should we extract a method, as the same code is used 5 times?

Reply:
Would be nice indeed but we need a bigger refactoring on graphs that will happen later. For now I don't think we want to take time on it, the whole file is a copy paste anyway...